docs/output.md

```diff
@@ -6,26 +6,71 @@ This document describes the output produced by the pipeline. Most of the plots a
 
```

Comment:
In the line above "Most of the plots are taken from the MultiQC report, which summarises results at the end of the pipeline." can be removed I think.